source/modules/MEI.xml

```diff
@@ -510,6 +510,25 @@
       </valList>
     </content>
   </macroSpec>
+  <macroSpec ident="data.CANCELACCID" module="MEI" type="dt">
+    <desc xml:lang="en">Indicates the location of the tonic in the circle of fifths.</desc>
+    <content>
+      <valList type="closed">
+        <valItem ident="none">
+          <desc xml:lang="en">Do not show cancellation accidentals.</desc>
```

Comment:
Maybe better to take the encoder's perspective than the renderer's: "Without cancellation accidentals."? Possibly with a "Written" or "Printed" in front of that.

Reply:
I meant that for all descriptions.